lib/print.rs

```diff
@@ -0,0 +1,3 @@
+pub fn print(msg: &String) {
+    print!("{}\n", msg)
+}
```

Comment:
```suggestion
use std::fmt::Debug;

pub fn debug<T>(value: &T) {
    println!("{:?}", value);
}
```

Taking arbitrary values that are debuggable is heaps more versatile easier to use than only taking strings

Reply:
Good idea.  All DDlog types should implement `Debug`, I think, so this gives a way to print anything.
`print` might still be useful though, as it forces you to convert values to string using DDlog's `to_stream()` function, which is kind of similar to Rust's `Display`.
I vote we keep both.